install.sh

```diff
@@ -20,4 +20,33 @@ echo "Host Entries..." >> $LOG
 echo "159.8.157.88 mcp-poc-uc.softlayer.com mcp-poc-uc" >> /etc/hosts
 echo "Done" >> $LOG
 
-. install_urbancode_agent.sh
+############################################################################
+## Install UrbanCode agent                                                ##
+############################################################################
+
+agent_download_path="https://github.com/garethjevans/provision-scripts/blob/master/ibm-ucd-agent.zip?raw=true"
+agent_download_name=`basename ${agent_download_path} | sed 's/\(^[a-z\]*-[a-z]*-[a-z]*.zip\).*$/\1/'`
+
+installation_dir=/opt/ibm-ucd/agent
+
+partial_ip=`ifconfig | tail -n +`ifconfig | grep -n eth1| awk -F':' '{print $1+1}'` | head -1 | awk '{print substr($2,6,10)}'`
+
+wget ${agent_download_path} -O /tmp/${agent_download_name} >> $LOG
+unzip /tmp/${agent_download_name}
+cd /tmp/${agent_download_name}-install
+
+./install_agent.sh << EOF
+${installation_dir}
+Y
+/opt/java8
+N
+${partial_ip}88
```

Comment:
i'm presuming this is the urbancode server? can we use 'mcp-poc-uc' instead? this should be in the host file